datahub-web-react/src/alchemy-components/components/Select/Nested/useSelectOption.ts

```diff
@@ -44,22 +46,8 @@ export default function useNestedOption<OptionType extends NestedSelectOption>({
     );
 
     const isSelected = useMemo(
-        () =>
-            !!selectedOptions.find((o) => o.value === option.value) ||
-            (!areParentsSelectable &&
-                !!option.isParent &&
-                !!selectableChildren.length &&
-                areAllChildrenSelected &&
-                !areAnyUnselectableChildrenUnexpanded),
-        [
-            selectedOptions,
-            areAllChildrenSelected,
-            areAnyUnselectableChildrenUnexpanded,
-            areParentsSelectable,
-            option.isParent,
-            option.value,
-            selectableChildren.length,
-        ],
+        () => !!selectedOptions.find((o) => o.value === option.value),
```

Comment:
Unfortunately I had to make these chnages because our current nested select didn't support single select correctly without showing parents as selected: 
![Uploading Screenshot 2025-09-03 at 5.50.28 PM.png…]()